nixos/modules/services/misc/taskchampion-sync-server.nix

```diff
@@ -45,6 +45,11 @@ in
         default = 14;
       };
     };
+    allow-client-ids = lib.mkOption {
+      description = "Client IDs to allow to connect";
+      type = types.listOf types.str;
```

Comment:
Maybe use one of integer types instead of `str`.

Reply:
> Maybe use one of integer types instead of `str`.

Are you sure about that? I didn't manage to figure it out myself from reading:

- https://github.com/GothenburgBitFactory/taskchampion-sync-server/pull/62

Reply:
Oh, id is a uuid.  Thanks for pointing out and sorry for this noise.  `str` is fine then.  (I do not use this software.)

Reply:
Thanks for reviewing anyway :).